json-schemas/src/attachments.json

```diff
@@ -0,0 +1,25 @@
+{
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "title": "Ably Attachment Event",
+  "type": "object",
+  "properties": {
+    "channelId": {
+      "type": "string",
+      "description": "The ID of the channel"
+    },
+    "channelName": {
+      "type": "string",
+      "description": "The name of the channel"
+    },
+    "qualifier": {
+      "type": "string",
+      "description": "The qualifier used in the channel spec"
+    },
+    "qualifierParam": {
+      "type": "string",
+      "description": "The parameter for the qualifier used in the channel spec"
+    }
```

Comment:
Perhaps include the requested [channel params](https://ably.com/docs/realtime/channels/channel-parameters/overview) and [mode](https://faqs.ably.com/what-are-channel-mode-flags-and-how-do-i-use-them)?

Reply:
Haven't included mode, how useful is that really?